README.md

```diff
@@ -50,15 +50,15 @@ Works with cactus, sugar cane, saplings, hoppers, dispensers and droppers.
 * Categories: `EXTRAS`, `EXPERIMENTAL`  
   
 ## chickenShearing
-Chickens can be sheared to get feathers. Beware! every time u shear a chicken, it gets damaged!  
+Chickens can be sheared to get feathers. Beware! every time you shear a chicken, it gets damaged!  
 Baby chickens can't be sheared.  
 * Type: `boolean`  
 * Default value: `false`  
 * Required options: `true`, `false`  
 * Categories: `EXTRAS`, `FEATURE`  
   
 ## clericsFarmWarts
-Clerics can warm nether farts.  
+Clerics can warm nether warts.  
```

Comment:
If this is ever going to be merged, then at least this needs to be done (as @altrisi) said.
```suggestion
Clerics can farm nether warts.
```

Reply:
and it also needs to be changed here:
https://github.com/gnembon/carpet-extra/blob/5bf3ffa2b167422fda9b743dd69ecfc0ce49db25/src/main/java/carpetextra/CarpetExtraSettings.java#L106